apple-app-site-association.json

```diff
@@ -1,6 +1,8 @@
 {
   "activitycontinuation": {
-    "apps": ["23KMWZ572J.net.artsy.artsy"]
+    "apps": [
+      "23KMWZ572J.net.artsy.artsy"
```

Comment:
this seems noisy, but also, was diff generated from `yarn build`, so I think it's worth checking in to avoid future needless diff.